ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -147,6 +147,44 @@ debug_pod() {
         oc logs "$pod"
 }
 
+# Wait for all pods of the app label to contain expected message
+#
+# Params:
+#   $1 - app label
+#   $2 - message to be present in the logs
+#   $3 - timeout (60)
+#   $4 - namespace (the current one)
+wait_for_app_pods_message() {
+	local app="$1"
+	local message="$2"
+	local timeout="$3"
+	local namespace="$4"
+	[ -z "$timeout" ] && timeout=60
+	[ -n "$namespace" ] && namespace=" -n $namespace "
+	local pod
+	local pods
+	local i
+	SECONDS=0
+	while :; do
+		pods=$(oc get pods -l app="$app" --no-headers=true $namespace | awk '{print $1}' | xargs)
+		[ -n "$pods" ] && break
+		if [ "$SECONDS" -gt "$timeout" ]; then
+			echo "Unable to find any pods for '-l app=\"$app\" in ${SECONDS}s"
```

Comment:
Not closing the opened `'`

Reply:
hi @ldoktor ,

Unless I am missing a magic feature of shell, I think it needs to increment `$SECONDS`, doesn't it?

Reply:
oh, I learned something new today! From bash's manpage:

```
 SECONDS
              Each  time  this parameter is referenced, it expands to the number of seconds since shell invocation.  If a value is assigned to SECONDS, the value returned upon subsequent references is the number of seconds since the
              assignment plus the value assigned.  The number of seconds at shell invocation and the current time are always determined by querying the system clock.  If SECONDS is unset, it loses its special properties, even if  it
              is subsequently reset.
```

Reply:
Bash is full of surprises ;-). Anyway I'll fix the missing `'` in v2